{"author": {}, "body": "[deleted]", "created_utc": 1586247412.0, "distinguished": null, "id": "fmo6pqf", "is_submitter": false, "link_id": "t3_fuy9mr", "parent_id": "t1_fmo5t7z", "score": 1, "stickied": false, "subreddit_id": "t5_2sjnq", "downs": 0, "total_awards_received": 0, "edited": false, "name": "t1_fmo6pqf", "collapsed": false, "permalink": "/r/TechNewsToday/comments/fuy9mr/no_5g_didnt_cause_the_coronavirus_pandemic/fmo6pqf/"}